mixins/selectable/test/selectable-mixin.test.js

```diff
@@ -0,0 +1,66 @@
+import { defineCE, expect, fixture, oneEvent } from '@open-wc/testing';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { SelectableMixin } from '../selectable-mixin.js';
+
```

Comment:
there could be 8 combinations of disabled, selectable, selected  states. 
All probably should be covered by testing expected corresponding dom state, and state change as result of change and click events

Reply:
I've adjusted the tests so that all of these combinations are taken into account - definitely important!